jaxrs-genericresponse/pom.xml

```diff
@@ -16,6 +16,10 @@
       <groupId>javax.ws.rs</groupId>
       <artifactId>javax.ws.rs-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
```

Comment:
This is needed because it was broken out with java 9/10?

Reply:
Yep. JDK 9 has removed jaxb-api from the core library.